assets/src/edit-story/components/canvas/canvasElementDropzone.js

```diff
@@ -88,7 +88,7 @@ function CanvasElementDropzone({ children }) {
   );
 
   return (
-    <Container onDrop={onDropHandler} onDragOver={onDragOverHandler}>
+    <Container onDrop={null} onDragOver={onDragOverHandler}>
```

Comment:
Just a test, I presume?

Reply:
Yes, this line demonstrates that the onDropHandler reference is a cause of re-rendering, that would need to be fixed in any final changes.